### PR TITLE
Component and layout base fixes 

### PR DESCRIPTION
**Pull request by alexfdezsauco**
This PR includes:
- Fixes IDisposableAnalyzer errors 
- Enqueues raise property changed events to wait for a render handler on BlorcComponentBase and BlorcLayoutComponentBase

### CODE REVIEW

File: src/Blorc.Core/Bindings/Binding.cs [hunk 1]
```diff
@@ -139,11 +141,11 @@ private void Initialize()
         protected override void Uninitialize()
         {
             _source.ValueChanged -= OnSourceValueChanged;
-            _source.Dispose();
+            _source?.Dispose();
```
Comment (GeertvanHorrik) [inline src/Blorc.Core/Bindings/Binding.cs:144]: if we nullcheck this, we should nullcheck the line before as well. It's better to write:

```
var source = _source;
if (source is not null)
{
    source.ValueChanged -= OnSourceValueChanged;
    source.Dispose();
}
```

File: src/Blorc.Core/Components/BlorcLayoutComponentBase.cs [hunk 2]
```diff
@@ -138,12 +139,18 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
...
+            {
+                propertyChangedAction();
+            }
         }
 
 
+
         protected override async Task OnInitializedAsync()
         {
-            await DocumentService.InjectBlorcCoreJsAsync();
```
Comment (GeertvanHorrik) [inline src/Blorc.Core/Components/BlorcLayoutComponentBase.cs:146]: No longer needed?

**Review by GeertvanHorrik — CHANGES_REQUESTED** [review]

File: src/Blorc.Core/Components/BlorcLayoutComponentBase.cs (hunk 2 above)
Comment (alexfdezsauco) [reply src/Blorc.Core/Components/BlorcLayoutComponentBase.cs:146]: No, must / can do such a call in the Program startup configuring the webAssemblyHost

image

or in the App just like this 

image

+

image

File: src/Blorc.Core.Example/Blorc.Core.Example.csproj [hunk 3]
```diff
@@ -5,6 +5,8 @@
     <UseBlazorWebAssembly>true</UseBlazorWebAssembly>
     <AssemblyName>Blorc.Core.Example</AssemblyName>
     <RootNamespace>Blorc.Example</RootNamespace>
+    <Nullable>enable</Nullable>
```
Comment (GeertvanHorrik) [inline src/Blorc.Core.Example/Blorc.Core.Example.csproj:8]: This is directory.build.nullable.props. Include thst one instead

**Review by GeertvanHorrik — CHANGES_REQUESTED** [review]

File: src/Blorc.Core.Example/Blorc.Core.Example.csproj [hunk 4]
```diff
@@ -16,14 +17,22 @@
     <!-- SonarQube requires a project guid -->
     <ProjectGuid>3D467864-4A28-485A-ADB2-80875974A273</ProjectGuid>
   </PropertyGroup>
-  
+
+  <PropertyGroup Condition="'$(Configuration)'=='Debug'">
```
Comment (GeertvanHorrik) [inline src/Blorc.Core.Example/Blorc.Core.Example.csproj:21]: This should be part of Directory.Build.implicit.props

File: src/Blorc.Core.Example/Blorc.Core.Example.csproj [hunk 5]
```diff
@@ -5,6 +5,7 @@
     <UseBlazorWebAssembly>true</UseBlazorWebAssembly>
     <AssemblyName>Blorc.Core.Example</AssemblyName>
     <RootNamespace>Blorc.Example</RootNamespace>
+    <ImplicitUsings>enable</ImplicitUsings>
```
Comment (GeertvanHorrik) [inline src/Blorc.Core.Example/Blorc.Core.Example.csproj:8]: not sure if I am ready for this. I read some other things about this, so let's create (yet another) shared.props (Directory.Build.implicitusings.props) so we can manage at scale if we need to patch something.

**Review by GeertvanHorrik — APPROVED** [review]

**Pull request merged**